contracts/core/extensions/CoreInternal.sol

```diff
@@ -92,6 +93,13 @@ contract CoreInternal is
         isValidFactory(_factoryAddress)
     {
         state.validFactories[_factoryAddress] = false;
+        for (uint256 i = 0; i < state.factories.length; i++) {
```

Comment:
Can we create an address[] helper method for this?

Reply:
I'll add to Trello and we can decide. Wanna get this merged within the scope of what it is.